src/main/java/it/grid/storm/info/du/DiskUsageService.java

```diff
@@ -0,0 +1,95 @@
+package it.grid.storm.info.du;
+
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.collect.Lists;
+
+import it.grid.storm.namespace.VirtualFSInterface;
+
+public class DiskUsageService {
+
+  private static final Logger log = LoggerFactory.getLogger(DiskUsageService.class);
+
+  private List<VirtualFSInterface> monitoredSAs;
+
+  private ScheduledExecutorService executor;
+  private boolean running;
+
+  public DiskUsageService(List<VirtualFSInterface> vfss) {
```

Comment:
The executor should be a dependency for this service, i.e. passed in as a parameter

Reply:
The executor is now a parameter with also the list of VFSs, the delay and the period